### DISCUSSION TITLE
'invalid interpolation' in docker-compose.yml?

### DISCUSSION THREAD

**Discussion by snerk4000** [mrlt8/docker-wyze-bridge#382] [Q&A]
the sample docker-compose.yml file, copy-pasted directly, fails with: 

`ERROR: Invalid interpolation format for "environment" option in service "wyze-bridge": "WYZE_EMAIL=${my@email.com}"`

... any suggestions?

**Comment by mrlt8** [comment]
Remove the `${}`. It should be `WYZE_EMAIL=my@email.com` and `WYZE_PASSWORD=mypassword`

I know it's a little confusing, but the `${}` is there for portainer compatibility.

**Reply by snerk4000** [reply]
@mrlt8 sorry to keep up on this but, i truly dont understand this one.  this is on ubuntu server, for some reason this worked fine on ubuntu desktop install...

`docker: error during connect: Post "http://docker:2375/v1.24/containers/create": dial tcp: lookup docker on 10.0.1.1:53: no such host.`

**Reply by mrlt8** [reply]
hmm, not sure about this one. Is that with docker-compose?

**Reply by snerk4000** [reply]
yes, same example file (with working auth info this time)

**Reply by mrlt8** [reply]
Do you have any other containers on your server? Can you try reinstalling/updating docker?

**Reply by snerk4000** [reply]
@mrlt8 everything has gone through apt update/upgrade.  cutting losses and putting ubuntu desktop back on the machine, it's not worth the frustration.
